docs/Bloom_Commands.md

```diff
@@ -121,30 +130,35 @@ modify this behavior.
 ### Parameters
 
 * **key**: The name of the filter
-* **CAPACITY**: If specified, should be followed by the desired capacity for the
+* **ITEMS**: Indicates the beginning of the items to be added to the filter. This
+    parameter must be specified.
+
+Optional parameters:
+
+* **NOCREATE**: If specified, indicates that the filter should not be created if
+    it does not already exist. If the filter does not yet exist, an error is
+    returned rather than creating it automatically. This may be used where a strict
+    separation between filter creation and filter addition is desired. It is an
+    error to specify `NOCREATE` together with either `CAPACITY` or `ERROR`.
+* **capacity**: If specified, should be followed by the desired `capacity` for the
     filter to be created. This parameter is ignored if the filter already exists.
     If the filter is automatically created and this parameter is absent, then the
-    default capacity (specified at the module-level) is used. See `BF.RESERVE`
+    default `capacity` (specified at the module-level) is used. See `BF.RESERVE`
     for more information on the impacts of this value.
-* **ERROR**: If specified, should be followed by the the error ratio of the newly
+* **error**: If specified, should be followed by the the `error` ratio of the newly
```

Comment:
Not sure I agree. All these arguments are for initialization. I will try to make it clear.

It makes little sense to have `NOCREATE` flag here since `insert` is a combination of `reserve+add` and with this flag, it is simply `add`.

Reply:
Text edited